### PR TITLE
Update Buddy Guidance.md

### PR DESCRIPTION
**Pull request by yaseminercan**
Adding guidance for buddies on what to do and how to help

### CODE REVIEW

**Review by Vikkigill — APPROVED** [review]

File: guides/Buddy Guidance.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # Buddy Guidance
 
-A buddy is a friend at Made Tech. In normal times, they would show you all the good food places near the office, and take you out for a coffee or lunch. In current times it might be a Hangouts coffee chat.
-Buddies proactively and frequently check in with a new starter to see how they are doing and if they need any help.
-Kind of like your designated person to direct questions to, outside of the People Team.
+You've been chosen to be a buddy to a new team member - Yay. Here's some guidance to help you find out what's expected.
+
+A buddy is a friend at Made Tech. Someone friendly who helps new team members connect and find their way, especially in their first 3 months with us when everything is new to them. 
+
+Being a buddy is a really important way to help a new person intergrate into Made Tech.
```
Comment (Mindez) [inline guides/Buddy Guidance.md:7]:
```suggestion
Being a buddy is a really important way to help a new person integrate into Made Tech. 
```

File: guides/Buddy Guidance.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
-Buddies proactively and frequently check in with a new starter to see how they are doing and if they need any help.
-Kind of like your designated person to direct questions to, outside of the People Team.
+You've been chosen to be a buddy to a new team member - Yay. Here's some guidance to help you find out what's expected.
+
+A buddy is a friend at Made Tech. Someone friendly who helps new team members connect and find their way, especially in their first 3 months with us when everything is new to them. 
+
+Being a buddy is a really important way to help a new person intergrate into Made Tech. 
+When we're back in the office, you will show them all the good lunch places near the office, take them out for a coffee or lunch and help them find their way in the local area. 
+
+As we're working remotely currently, most or all of your intereactions will happen on video calls which we call 'coffee chats'.
```
Comment (Mindez) [inline guides/Buddy Guidance.md:10]:
```suggestion
As we're working remotely currently, most or all of your interactions will happen on video calls which we call 'coffee chats'.
```

File: guides/Buddy Guidance.md [hunk 3]
```diff
@@ -1,5 +1,19 @@
...
+
+## Buddies 
+- proactively and frequently check in with a new starter to see how they are doing. In the first 3 months you may check in 2-3 per month. After that time you can work out what suits you both, i.e. monthly catchup/remote lunch
```
Comment (Mindez) [inline guides/Buddy Guidance.md:13]:
```suggestion
- proactively and frequently check in with a new starter to see how they are doing. In the first 3 months you may check in 2-3 times per month. After that time you can work out what suits you both, i.e. monthly catchup/remote lunch
```

**Comment by Mindez** [comment]
Just passing by, thought I'd contribute a few friendly spelling fixes!

**Comment by yaseminercan** [comment]
merging

**Pull request merged**